samples/net/dsa/src/main.c

```diff
@@ -24,32 +24,69 @@ static void dsa_iface_find_cb(struct net_if *iface, void *user_data)
 		return;
 	}
 
-	if (net_eth_get_hw_capabilities(iface) & ETHERNET_DSA_MASTER_PORT) {
-		if (ifaces->master == NULL) {
-			ifaces->master = iface;
+	if (net_eth_get_hw_capabilities(iface) & ETHERNET_DSA_CONDUIT_PORT) {
+		if (ifaces->conduit == NULL) {
+			ifaces->conduit = iface;
 
-			/* Get slave interfaces */
+			/* Get user interfaces */
 			for (int i = 0; i < ARRAY_SIZE(ifaces->lan); i++) {
-				struct net_if *slave = dsa_get_slave_port(iface, i);
+#if defined(CONFIG_NET_DSA_DEPRECATED)
+				struct net_if *user = dsa_get_slave_port(iface, i);
+#else
+				struct net_if *user = dsa_user_get_iface(iface, i);
+#endif
 
-				if (slave == NULL) {
+				if (user == NULL) {
 					continue;
 				}
-				LOG_INF("Slave interface %d found.", i);
+				LOG_INF("[%d] User interface %d found.", i,
+					net_if_get_by_iface(user));
 
-				ifaces->lan[i] = slave;
+				ifaces->lan[i] = user;
 			}
 			return;
 		}
 	}
 }
 
+#if defined(CONFIG_NET_MGMT_EVENT)
+#define EVENT_MASK (NET_EVENT_IF_UP)
+static struct net_mgmt_event_callback mgmt_cb;
+
+static void event_handler(struct net_mgmt_event_callback *cb,
+			  uint32_t mgmt_event, struct net_if *iface)
+{
+	ARG_UNUSED(iface);
+	ARG_UNUSED(cb);
+
+	if ((mgmt_event & EVENT_MASK) != mgmt_event) {
+		return;
+	}
```

Comment:
This check is not really needed as you are checking the event few lines below.

Reply:
Will remove, thanks.